docs/resources/recognition.md

```diff
@@ -0,0 +1,46 @@
+# Recognizing Contributions
+
+There are a variety of ways that we would like to recognize community contributions to hackweeks. 

```

Comment:
Right now we only show the one way (adding self to the website).

Reply:
Maybe we should list out the three presented options from this page under this intro sentence as:
* Website Presence
* Online Tutorials
* Referrals and Feedback

The website presence takes the bulk of the content, but we are really presenting three options here